srcdoc/configure/access_control.mt

```diff
@@ -241,4 +241,33 @@ paths:
 EOT
 ?>
 
+<h2 id="how-to" class="section-head">How-To</h2>
+
+<h3 id="cidr">IP Address Matching Against CIDR list</h3>
```

Comment:
Possibly change the title to "Matching IP Address Blocks?".

Note that CIDR is an abbreviation of Classless Inter-Domain Routing. Are we routing packets?